### PR TITLE
0.7.10-->0.7.11

### PR DESCRIPTION
**Pull request by yeelali14**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=GAyD83s)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +75,17 @@ I'm better off when I hit the bottom
 
 Lady gaga the queen
 
+We're headed for a cliff
+And in the free fall I will realize
+I'm better off when I hit the bottom
+And the worst part is
+Before it gets any better
```
Comment (EladKohavi) [inline README.md:82]: okok

File: README.md [hunk 2]
```diff
@@ -75,3 +75,17 @@ I'm better off when I hit the bottom
...
+We're headed for a cliff
+And in the free fall I will realize
+I'm better off when I hit the bottom
+
```
Comment (EladKohavi) [inline README.md:86]: fg

File: README.md [hunk 3]
```diff
@@ -75,3 +75,17 @@ I'm better off when I hit the bottom
...
+And in the free fall I will realize
+I'm better off when I hit the bottom
+
+Lady gaga the queen
+
```
Comment (EladKohavi) [inline README.md:91]: hfd

File: README.md [hunk 4]
```diff
@@ -75,3 +75,17 @@ I'm better off when I hit the bottom
 
 Lady gaga the queen
 
+We're headed for a cliff
+And in the free fall I will realize
```
Comment (EladKohavi) [inline README.md:79]: fgh

**Review by EladKohavi — CHANGES_REQUESTED** [review]
h

File: README.md [hunk 5]
```diff
@@ -75,3 +75,17 @@ I'm better off when I hit the bottom
 
 Lady gaga the queen
 
+We're headed for a cliff
```
Comment (EladKohavi) [inline README.md:78]: HOLA

File: README.md [hunk 6]
```diff
@@ -75,3 +75,17 @@ I'm better off when I hit the bottom
 
 Lady gaga the queen
```
Comment (EladKohavi) [inline README.md:77]: jgh

**Review by ShakedZrihen — COMMENTED** [review]
ffff

**Review by EladKohavi — CHANGES_REQUESTED** [review]
hfgd

**Pull request closed without merging**